src/vs/vscode.proposed.d.ts

```diff
@@ -2754,4 +2754,15 @@ declare module 'vscode' {
 	}
 
 	//#endregion
+
+	//#region https://github.com/microsoft/vscode/issues/129053
+
+	export namespace env {
+		/**
+		 * The environment in which the app is embedded in. i.e. Desktop, Codespaces, Web.
```

Comment:
Other changes in this commit suggest that the string values aren't capitalised.

Reply:
Thank you that is correct they are lowercase, I'm pushed https://github.com/microsoft/vscode/commit/5b9f4ad720994cd0fa9f3bae0b956290ee9929e5